ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/HelperFunctions.kt

```diff
@@ -0,0 +1,9 @@
+package com.revenuecat.purchases.ui.revenuecatui.helpers
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.platform.LocalInspectionMode
+
+@Composable
+@ReadOnlyComposable
+internal fun isInPreviewMode() = LocalInspectionMode.current
```

Comment:
Can this be a lazy constant, so we don't evaluate it multiple times?

Reply:
Hmm I'm not sure you can declare a constant and mark it as `@Composable`, which is needed to use `LocalInspectionMode.current`. In any case, this composable shouldn't be recomputed unless the local composition value changes thanks to `@ReadOnlyComposable`.

Reply:
Oooh nice gotcha